src/client/public/sign-for-beta.html

```diff
@@ -90,55 +90,45 @@ <h2 class="who-we-are-title">
 						<form class="project-form">
 							<div class="project-form-field">
 								<label for="name">Full Name <span style="color: red">*</span></label>
-								<div class="personal-row" style="align-items: center;">
-									<input class="pick-input" name="name" id="name" type="text" placeholder="" required>
-									<span class="error-symbol" style="display: none">!</span>
-									<span class="good-symbol" style="display: none"><svg width="22" height="27" viewBox="0 0 22 27" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M2 13.0377L10.7568 23L20 1" stroke="white" stroke-width="4"/>
-</svg>
-</span>
+								<div id="container1" class="personal-coloum input-continer" style="align-items: center;">
+									<input id="input1" name="name" type="text" placeholder="" required>
+									<span class="error-message" id="error-1"> Please enter your full name</span>
 								</div>
 							</div>
 							<div class="project-form-field">
 								<label for="name">Email address  <span style="color: red">*</span></label>
-								<div class="personal-row" style="align-items: center;">
-									<input class="error-input"  name="email" id="email" type="text" placeholder="" required>
-									<span class="error-symbol">!</span>
-									<span class="good-symbol" style="display: none"><svg width="22" height="27" viewBox="0 0 22 27" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M2 13.0377L10.7568 23L20 1" stroke="white" stroke-width="4"/>
-</svg>
-</span>
+								<div id="container2"  class="personal-coloum input-continer" style="align-items: center;">
```

Comment:
Don't use "style" in HTML tags!!!
Preferably, add classes and edit style in main.css